test/cluster/test_tablets2.py

```diff
@@ -1808,3 +1808,72 @@ def move_sstables_to_upload(table_dir: str):
         await load_and_stream_task
 
         await check(ks)
+
+@pytest.mark.asyncio
+@skip_mode('release', 'error injections are not supported in release mode')
+async def test_timed_out_reader_after_cleanup(manager: ManagerClient):
+    logger.info("Bootstrapping cluster")
+    cmdline = [
+        '--logger-log-level', 'storage_service=debug',
+        '--logger-log-level', 'raft_topology=debug',
+        '--range-request-timeout-in-ms', '1000', # shorten time coordinator abandon the request, releasing erm
+        '--read-request-timeout-in-ms', '1000',
+        '--abort-on-internal-error', 'true',
```

Comment:
nit: `--abort-on-internal-error` is redundant -- it's turned on by default in
`SCYLLA_CMDLINE_OPTIONS`

Reply:
Agree, ok.